client/src/components/Accordion/Accordions.js

```diff
@@ -24,7 +24,7 @@ function FirstChild(props) {
   return childrenArray[0] || null;
 }
 
-const defaultMaxHeight = "300px";
+const defaultMaxHeight = "500px";
```

Comment:
Just some food for thought for the next person reviewing. Is there a reason we shouldn't have this at `80vh` or greater?

As far as I know, this height is only used for the `Transition` component/the actual animation occurring. With that being said, we know that any accordion according to the following class `StatelessPullDownAccordion` caps the child height (before overflowing) to `80vh`. Thus our default animation height would never exceed `80vh` anyways (well we'd have to add the accordion header and footer size as well).

With that being said, we could probably get rid of this `defaultMaxHeight` like so (untested, may need slight adjustment if we choose to do this):

```
const StatelessPullDown = ({max_height, ...existing_props}) => {
  <existing divs>
    <AccordionEnterExit
          className="pull-down-accordion-body"
          style={{ paddingTop: "5px" }}
          expandDuration={600}
          collapseDuration={600}
          max_height={max_height}
        >
          <div style={{ maxHeight: max_height, overflowY: "auto" }}>{children}</div>
        </AccordionEnterExit>
  </existing divs>
}
StatelessPullDownAccordion.defaultProps = {
  max_height: "80vh"
}
```

Now I guess we didn't really remove `defaultMaxHeight` in a sense, but we get the added control over the max content height of the accordion so it could be smaller or larger if we really wanted it to be the case

Reply:
yeah `defaultMaxHeight` can prob go, and have defaultProps 80vh for both `StatelessPulldownAccordion` and also `AccordionEnterExit` (because it seems like it's used in explorer_common as well?)

Reply:
I just pushed a change that (I think?) properly removes the `defaultMaxHeight` variable and uses the new `max_height` variable.